configs/2020_06_23_ocp1k/schnet.yml

```diff
@@ -0,0 +1,24 @@
+includes:
+- configs/2020_06_23_ocp1k/base.yml
+
+model:
+  name: schnet
+  hidden_channels: 128
+  num_filters: 128
+  num_interactions: 3
+  num_gaussians: 200
+  cutoff: 6.0
+
+optim:
+  batch_size: 256
```

Comment:
This value is too high, crashed my run for memory reasons. 128 is running fine, maybe set to 64 to be safe.